src/sage/schemes/weighted_projective/__init__.py

```diff
@@ -0,0 +1 @@
+# Here so that cython creates the correct module name
```

Comment:
this file isn't supposed to be committed into git, add it to `# Meson temporary files` section in `.gitignore` instead